unittests/svnn_ibc_tests.cpp

```diff
@@ -574,12 +588,18 @@ BOOST_AUTO_TEST_SUITE(svnn_ibc)
       // The QC provided to prove this also proves a commitment from finalizers to this policy, so the smart contract can accept it.
       action_trace check_heavy_proof_3_trace = cluster.node0.push_action("ibc"_n, "checkproof"_n, "ibc"_n, heavy_proof_3)->action_traces[0];
 
+      BOOST_CHECK(true);

```

Comment:
???

Reply:
Sorry for that. I added these checkpoints to verify which push_action calls were failing (ctest lets you know which was the last valid checkpoint), and forgot to remove them. I removed them now